examples/gce/cluster.yaml

```diff
@@ -24,6 +24,10 @@ spec:
             configmap: repo
             key: docker-ce.repo
           destination: /etc/yum.repos.d/docker-ce.repo
+        - source:
+            configmap: repo
+            key: cloud-google-com.gpg.b64
+          destination: /tmp/cloud-google-com.gpg.b64
```

Comment:
Where does this get referenced?

Reply:
This is referenced here https://github.com/weaveworks/wksctl/blob/master/pkg/plan/recipe/install_plans.go#L185 but it was missing from the example config so it wasn't working

Reply:
Thanks; that's an excellent example of something that can go in the commit message so the reviewer doesn't need to puzzle about it.